genie-agent/src/main/java/com/netflix/genie/agent/execution/ExecutionContextImpl.java

```diff
@@ -225,6 +227,24 @@ public void setClaimedJobId(@NotBlank final String jobId) {
         setIfNullOrTrow(jobId, claimedJobIdRef);
```

Comment:
`setIfNullOrTrow` misspelled

Reply:
Ah thanks for spotting this.
I already have other commits in the pipe that modify this class, so I'll fix this as part of that, rather than in this PR.